src/app/globals.css

```
@@ -262,20 +262,20 @@ html {
     font-size: 16px; /* Ensure consistent paragraph font size */
   }
 
-  /* Force code blocks to wrap properly */
+  /* Force block code (<pre> and its <code>) to wrap; inline <code> keeps
+     component-level `whitespace-normal` so it breaks at word boundaries only */
   .blog-content pre,
-  .blog-content code {
+  .blog-content pre code {
     white-space: pre-wrap;
     overflow-wrap: break-word;
     word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)